cookbooks/fb_bookworm/README.md

```diff
@@ -0,0 +1,29 @@
+fb_bookworm
```

Comment:
This README doesn't have some of the sections that the internal README parser would expect for the autogenerated docs.

Reply:
https://github.com/facebook/chef-cookbooks/commit/b74bf2f119b36b56510d3e8f24b0266dc5b3825d adds several more sections, which puts it a little more in line with the other cookbooks, and should see more polish in the coming weeks as the program's sharp edges are buffed out